fluss-connectors/fluss-connector-flink/src/test/java/com/alibaba/fluss/connector/flink/sink/FlinkTableSinkITCase.java

```diff
@@ -356,6 +357,39 @@ void testPartialUpsert() throws Exception {
         assertResultsIgnoreOrder(rowIter, expectedRows, true);
     }
 
+    @Test
+    void testFirstRowMergeEngine() throws Exception {
+        tEnv.executeSql(
+                "create table first_row_source (a int not null primary key not enforced,"
+                        + " b string) with('table.merge-engine' = 'first_row')");
+        tEnv.executeSql("create table first_row_sink (a int, b string)");
+
+        // insert from first_row_source to first_row_sink
+        JobClient insertJobClient =
+                tEnv.executeSql("insert into first_row_sink select * from first_row_source")
```

Comment:
Is this used for verifying append-only strema? If yes, 

1. add the motivation comment, `insert from first_row_source to first_row_sink` is a useless comment as we all know from the code. 
2. the sink table name would be better to `log_sink` 
3. read from the `log_sink` and verify the result.